### DISCUSSION TITLE
What camera settings should I use to connect Swatch to Frigate snapshots?

### DISCUSSION THREAD

**Discussion by heisenberg2980** [NickM-27/swatch#57] [Q&A] [answered]
Is there a way to connect Swatch to Frigate so that when Frigate detect movement on a camera it sends the snapshot to Swatch?

Trying to setup Swatch but I am not sure what url should be used to replace "http://ip.ad.dr.ess/jpg"

**Comment by NickM-27** [comment, marked as answer]
No that is not supported, swatch can just pull a snapshot on a regular interval from frigate.

You'd want to use `http://frigate_ip:5000/api/camera_name/latest.jpg`

**Reply by heisenberg2980** [reply]
Thanks for the quick response, that is exactly what I was looking for, and thank you as well for this incredible project which complements Frigate perfectly to detect objects by colour that would be very difficult to achieve with Frigate
